app/locales/ar.json

```diff
@@ -54,7 +54,7 @@
     "choose_provider_subtitle": "لتكون على علم بالتعرض والانتشار ، ستحتاج إلى الاشتراك في هيئة صحية",
     "choose_provider_title": "اختر الهيئة الصحية",
     "commitment": "التزام",
-    "commitment_para": "تسجل المسارات الآمنة بشكل آمن وتتحقق من تفاعلك مع الأشخاص الذين يستخدمون موقعك. لن تترك بياناتك هاتفك أبدًا بدون موافقتك",
+    "commitment_para": "المسارات الآمنة هي حركة عالمية لتطوير أدوات مجانية ومفتوحة المصدر وخصوصية حسب التصميم للمقيمين ومسؤولي الصحة العامة والمجتمعات الأكبر من أجل تسوية منحنى COVID-19 وتقليل الخوف ومنع استجابة الدولة للمراقبة الوباء \n\nلمزيد من المعلومات، يرجى زيارة الموقع",
```

Comment:
undo all these changes, update only `en.json`.

We use another platform for translations